pkg/subctl/cmd/gather/broker.go

```diff
@@ -0,0 +1,33 @@
+/*
+© 2021 Red Hat, Inc. and others.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package gather
+
+import (
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+const brokerNamespace = "submariner-k8s-broker"
```

Comment:
The broker isn't on a fixed place, 

You can probably benefit from this here (to get the rest config to the cluster), and also from the submariner object you can get the namespace of the broker (see https://github.com/submariner-io/submariner-operator/blob/2efe168a8015624ec875c6d363e781c59639d039/apis/submariner/v1alpha1/submariner_types.go#L41 ) 

https://github.com/submariner-io/submariner-operator/pull/1190/files